db/migrate/20180927135248_create_javascript_caches.rb

```diff
@@ -0,0 +1,10 @@
+class CreateJavascriptCaches < ActiveRecord::Migration[5.2]
+  def change
+    create_table :javascript_caches do |t|
+      t.references :theme_field, null: false
+      t.string :digest, null: true, index: true
+      t.text :content, null: false
+      t.timestamps
+    end
```

Comment:
There are no indexes on this table. It looks like it's mainly queried by `digest` so it should have an index on that, probably a unique index?

Reply:
There is one index on `theme_field_id` (added by `t.references` helper), and another on `digest` (added by `index: true`).

`digest` is calculated entirely based on `content`, so it may not be unique (one common case: snippets copied from a Meta post). We could calculate it with `"#{id}_#{content}"` to make it "unique", do you think it's necessary?

Reply:
Oh my bad, I did not see the `index: true` - In fact I didn't know you could add one that way. It's fine if it's not unique, I was wrong about that.